### PR TITLE
Protect: Add notifications

### PR DESCRIPTION
**Pull request by leogermani**
#### Changes proposed in this Pull Request:

* Adds notifications to the menu item and the admin bar

#### Jetpack product discussion

N/A

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Activate Protect
* Set the response_type to `complete_green`
* Confirm you don't see anything in the Protect menu item neither in the admin bar
* Set the response_type constant to `complete`
* Confirm you see notifications in the menu item and in the admin bar
* Confirm the numbers displayed match the total number of vulneratbilities
* Confirm that clicking on the admin bar takes you to the Protect page

Captura de tela de 2022-04-27 13-06-17

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1202064326332574

### CODE REVIEW

**Comment by renatoagds** [comment]
What do you think about calling this folder `assets` instead of `css`?

**Review by renatoagds — COMMENTED** [review]
There's some lint errors happening

**Comment by leogermani** [comment]
FIxed

File: projects/plugins/protect/src/class-jetpack-protect.php [hunk 1]
```diff
@@ -28,12 +28,27 @@ class Jetpack_Protect {
...
+	/**
+	 * Initialize the plugin
+	 *
+	 * @return void
+	 */
+	public function init() {
 		// Set up the REST authentication hooks.
 		Connection_Rest_Authentication::init();
 
+		$total_vuls = Status::get_total_vulnerabilities();
```
Comment (retrofox) [inline projects/plugins/protect/src/class-jetpack-protect.php:43]: Getting Fatal here

```
[28-Apr-2022 14:45:20 UTC] PHP Fatal error:  Uncaught Error: Class "Status" not found in /usr/local/src/jetpack-monorepo/projects/plugins/protect/src/class-jetpack-protect.php:42
Stack trace:
```

```suggestion
		$total_vuls = Protect_Status::get_total_vulnerabilities();
```

**Review by retrofox — CHANGES_REQUESTED** [review]
I'm requesting changes because of
* PHP Fatals. Commits suggested.
* The icon doesn't look very well compared with the rest
image
image

**Comment by leogermani** [comment]
@retrofox Can we merge this and create another issue to fix the icon aligment?

**Review by retrofox — APPROVED** [review]
LGTM

**Review by retrofox — APPROVED** [review]
LGTM

**Pull request merged**